cypress/e2e/Layout/Editor/layout_editor_options.cy.js

```diff
@@ -0,0 +1,337 @@
+            });
+        });
+    });
+
+    it('should enable tooltips', function() {
+        cy.wait(1000);
+        cy.openOptionsMenu();
+
+        cy.intercept('POST', '/user/pref').as('updatePreferences');
+
+        // Enable tooltips
+        // Check the current state of the tooltips checkbox
+        cy.get('#displayTooltips').then(($checkbox) => {
+            if (!$checkbox.is(':checked')) {
+                // Check the checkbox if it is currently unchecked
+                cy.wrap($checkbox).click();
+                cy.wait('@updatePreferences');
+
+                // Confirm the checkbox is checked
+                cy.get('#displayTooltips').should('be.checked');
+            } 
+        });
+
+        // Verify that tooltips are present
+        cy.get('.navbar-nav .btn-menu-option[data-toggle="tooltip"]').each(($element) => {
+            // Trigger hover to show tooltip
+            cy.wrap($element).trigger('mouseenter');
+        
+            // Check that the tooltip is visible for each button
+            cy.get('.tooltip').should('be.visible'); // Expect tooltip to be present
+        });
+
+    });
+
+    it('should disable tooltips', function() {
+        cy.wait(1000);
+        cy.openOptionsMenu();
+
+        cy.intercept('POST', '/user/pref').as('updatePreferences');
+
+        // Disable tooltips
+        // Check the current state of the tooltips checkbox
+        cy.get('#displayTooltips').then(($checkbox) => {
+            if ($checkbox.is(':checked')) {
+                // Uncheck the checkbox if it is currently checked
+                cy.wrap($checkbox).click();
+                cy.wait('@updatePreferences');
+
+                // Confirm the checkbox is now unchecked
+                cy.get('#displayTooltips').should('not.be.checked');
+            } 
+        });
+
+        // Verify that tooltips are gone
+        cy.get('.navbar-nav .btn-menu-option[data-toggle="tooltip"]').each(($element) => {
+            cy.wrap($element).trigger('mouseenter'); // Trigger hover to show tooltip        
+            cy.get('.tooltip').should('not.exist'); // Check if tooltip is gone for each button on the toolbox
+        });
+
+    });
+
+    it('should enable delete confirmation', function() {
+        cy.wait(1000);
+        cy.openOptionsMenu();
+
+        cy.intercept('POST', '/user/pref').as('updatePreferences');
+
+        // Check the current state of the delete confirmation checkbox
+        cy.get('#deleteConfirmation').then(($checkbox) => {
+            if (!$checkbox.is(':checked')) {
+                // Check the checkbox if it is currently unchecked
+                cy.wrap($checkbox).click();
+                cy.wait('@updatePreferences');
+
+                // Confirm the checkbox is checked
+                cy.get('#deleteConfirmation').should('be.checked');
+            } 
+        });
+
+        // Add an element then attempt to delete
+        cy.openToolbarMenu(0);
```

Comment:
You can modify the openToolbarMenu command to accept an additional parameter that indicates whether to wait for the toolbar preferences to load. If load is set to false, the command will skip the following two lines:

```
  // Wait for the toolbar to reload when getting prefs at start, based on the load parameter
  if (load) {
    cy.wait('@toolbarPrefsLoad');
    cy.wait('@editorPrefsLoad');
  }
```